index_api-v2.yaml

```diff
@@ -0,0 +1,722 @@
+openapi: 3.0.0
+info:
+  version: 2.0.0
+  title: Murmurations Index API
+  description: API schema for the Murmurations Index
+servers:
+  - description: Murmurations Testing Platform
+    url: https://test-index.murmurations.network/v2
+tags:
+  - name: Common Endpoints
+  - name: Node Endpoints
+  - name: Aggregator Endpoints
+paths:
+  /ping:
+    get:
+      tags:
+        - Common Endpoints
+      summary: Ping the index
+      description: |
+        Check if the index is live
+      responses:
+        200:
+          description: OK
+  /validate:
+    post:
+      tags:
+        - Node Endpoints
+      summary: Validate a node profile
+      description: A node operator may want to validate the node profile will be accepted by the index before posting it to the node's website. This endpoint enables such a validation check.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/ValidateNode"
+            example:
+              linked_schemas:
+                - "demo_schema-v1"
+              name: "Some Org"
+      responses:
+        200:
+          description: OK
+        400:
```

Comment:
See comment above re using 400 for this...

Reply:
Noted, and fixed. See my next commit.